src/index.html

```diff
@@ -1,13 +1,709 @@
+                The Met Fifth Avenue <br>
+                +1 212-535-7710 <br>
+                The Met Cloisters <br>
+                +1 212-923-3700
+              </div>
+            </a>
+
+            <a href="#menu" class=" icon icon--menu"></a>
+          </nav>
+
+          <h1
+            class="header__title"
+            data-aos="zoom-in"
+            data-aos-duration="2000"
+          >
+            Welcome
+            <br>
+            to The MET
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top">
+            <a href="#" class="logo">
+              <img
+                class="logo__image"
+                src="./images/logo.svg"
+                alt="logo"
+              >
+            </a>
+
+            <a href="#" class=" icon icon--cross"></a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">
+                Home
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#about" class="menu__link">
+                About us
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#nowonview" class="menu__link">
+                Now on view
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#explore" class="menu__link">
+                Explore online
+              </a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">
+                Contacts
+              </a>
+            </li>
+          </ul>
+
+          <a
+            href="tel:+1 212-535-7710"
+            class="menu__call menu__call--phone">
+            The Met Fifth Avenue<br>
+            +1 212-535-7710<br>
+          </a>
+
+          <a
+            href="tel:+1 212-923-3700"
+            class="menu__call menu__call--phone">
+            The Met Cloisters <br>
+            +1 212-923-3700
+          </a>
+
+          <a
+            href="tel:+1 212-535-7710"
+            class="menu__call menu__call--action">
+            Call to order
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section class="page__section about" id="about">
+        <div class="container">
+          <div
+            class="grid
+            grid--desktop
+            grid--tablet"
+          >
+            <div
+              class="grid__item
+              grid__item--tablet-1-3
+              grid__item--desktop-1-5"
+              data-aos="fade-up"
+              data-aos-duration="2000"
+            >
+              <p class="about__pre-title">
+                Plan your visit
+              </p>
+
+              <h2 class="page__section-title">
+                Museum hours
```

Comment:
According to the mockup the size of titles is big starting from tablet version, but it stays small on your solution.

![image](https://user-images.githubusercontent.com/78492113/218102136-ddb072d6-ddff-4fae-a9d7-27fc8a72cfef.png)

Reply:
Thanks for review) Everything was fixed)